docs/guides/documentation-style.stories.mdx

```diff
@@ -0,0 +1,87 @@
+import { Meta } from "@storybook/addon-docs";
+import showDo from "./assets/Atlantis_Docs_Show_Do.png";
+import showDont from "./assets/Atlantis_Docs_Show_Dont.png";
+
+<Meta title="Guides/Documentation styleguide" />
+
+# Documentation styleguide
+
+<br />
+
+## Purpose
```

Comment:
Honestly not even sure if we need this purpose section

Reply:
Yeah it's not super useful.